examples/abitest/module_0/rust/tests/integration_test.rs

```diff
@@ -161,6 +156,23 @@ async fn test_leaks() {
             want_channels += result.channel_change;
         }
 
+        let (after_nodes, after_channels) = runtime.object_counts();
+        // If the values don't match the expected values it could be due to a race condition where a
+        // backend node closed its orphaned channeal a bit later than expected, so we wait 5 seconds
+        // and calculate the values again.
+        let (after_nodes, after_channels) =
+            if after_nodes != want_nodes || after_channels != want_channels {
+                sleep(Duration::from_secs(5)).await;
+                runtime.object_counts()
+            } else {
+                (after_nodes, after_channels)
+            };
+
+        info!(
+            "Counts change from test: Nodes={} => {}, Channels={} => {}",
+            before_nodes, after_nodes, before_channels, after_channels
+        );
```

Comment:
I guess if this become a more common pattern then we may want a higher level fn like `retry<T>(f: Fn() -> T) -> Result<T, RetryErr>`.